runtime-sdk/modules/evm/src/precompile/confidential.rs

```diff
@@ -682,7 +685,7 @@ mod test {
 
         // Unsupported method.
         let params = ethabi::encode(&[
-            Token::Uint(6.into()), // sr25519 is not yet supported.
+            Token::Uint(10.into()), // 10 is invalid
```

Comment:
Consider either using the lowest invalid method number (which is 9 at the moment, I believe) or a larger value that won't be valid anytime soon (like 255)?

Reply:
This should actually be removed as an invalid method is tested above this. This was for a valid method but unsupported in the precompiles and with sr25519 added there is currently no such method.